### PR TITLE
sfizz: 0.5.1 -> 1.1.1

### PR DESCRIPTION
**Pull request by hirenashah**
###### Motivation for this change
v1.1.1 is available, but the package has not been updated since v0.5.1.

###### Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [X] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [X] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by magnetophon** [comment]
I've also ried to update, but ran into https://github.com/sfztools/sfizz/issues/505.
This PR has the same issue: when zenity is not in the path, the plugin isn't able to call it, even with the patching.

File: pkgs/applications/audio/sfizz/default.nix [hunk 1]
```diff
@@ -37,18 +35,18 @@ stdenv.mkDerivation rec {
...
     gnome.zenity
     freetype
+    pango
   ];
   nativeBuildInputs = [ cmake pkg-config ];
 
   postPatch = ''
-  substituteInPlace editor/external/vstgui4/vstgui/lib/platform/linux/x11fileselector.cpp \
-    --replace '"/usr/bin/zenity' '"${gnome.zenity}/bin/zenity'
+    substituteInPlace plugins/editor/external/vstgui4/vstgui/lib/platform/linux/x11fileselector.cpp \
```
Comment (hirenashah) [inline pkgs/applications/audio/sfizz/default.nix:43]: > I've also ried to update, but ran into [sfztools/sfizz#505 (comment)](https://github.com/sfztools/sfizz/issues/505#issuecomment-988868693). This PR has the same issue: when zenity is not in the path, the plugin isn't able to call it, even with the patching.

I had zenity available system wide which is why I didn't notice the problem. I think I've somehow fixed the issue with the file dialog not showing but the fix on this line doesn't seem correct.

Comment (magnetophon) [reply pkgs/applications/audio/sfizz/default.nix:43]: I still don't get the dialog, unless I have zenity system wide.

> but the fix on this line doesn't seem correct.

Not sure what you mean by that.

Comment (hirenashah) [reply pkgs/applications/audio/sfizz/default.nix:43]: Did you try this branch? I don't have zenity available system wide anymore and the change I made on line 43-44 seems to have resolved it for me.

Comment (magnetophon) [reply pkgs/applications/audio/sfizz/default.nix:43]: I'm sorry; I thought I had tried the correct binary, built from this branch, but it seems like I didn't.
When I tried again just now, everything works.

**Review by magnetophon — APPROVED** [review]

**Pull request merged**